public/app/plugins/datasource/cloudwatch/components/ConfigEditor.test.tsx

```diff
@@ -104,53 +93,75 @@ describe('Render', () => {
     };
     jest.resetAllMocks();
     putMock.mockImplementation(async () => ({ datasource: setupMockedDataSource().datasource }));
+    datasource.api.describeLogGroups = jest.fn().mockResolvedValue(['logGroup-foo', 'logGroup-bar'].map(toOption));
+    datasource.api.getRegions = jest.fn().mockResolvedValue([
+      {
+        label: 'ap-east-1',
+        value: 'ap-east-1',
+      },
+    ]);
+    datasource.getActualRegion = jest.fn().mockReturnValue('ap-east-1');
+    datasource.getVariables = jest.fn().mockReturnValue([]);
   });
 
-  it('should render component without blowing up', () => {
-    expect(() => setup()).not.toThrow();
-  });
-
-  it('should disable access key id field', () => {
+  it('should disable access key id field', async () => {
     setup({
       secureJsonFields: {
         secretKey: true,
       },
     });
-    expect(screen.getByPlaceholderText('Configured')).toBeDisabled();
+    await waitFor(async () => expect(screen.getByPlaceholderText('Configured')).toBeDisabled());
   });
 
-  it('should show credentials profile name field', () => {
+  it('should show credentials profile name field', async () => {
     setup({
       jsonData: {
         authType: AwsAuthType.Credentials,
       },
     });
-    expect(screen.getByLabelText('Credentials Profile Name')).toBeInTheDocument();
+    await waitFor(async () => expect(screen.getByLabelText('Credentials Profile Name')).toBeInTheDocument());
   });
 
-  it('should show access key and secret access key fields', () => {
+  it('should show access key and secret access key fields', async () => {
     setup({
       jsonData: {
         authType: AwsAuthType.Keys,
       },
     });
-    expect(screen.getByLabelText('Access Key ID')).toBeInTheDocument();
-    expect(screen.getByLabelText('Secret Access Key')).toBeInTheDocument();
+    await waitFor(async () => {
+      expect(screen.getByLabelText('Access Key ID')).toBeInTheDocument();
+      expect(screen.getByLabelText('Secret Access Key')).toBeInTheDocument();
+    });
   });
 
-  it('should show arn role field', () => {
+  it('should show arn role field', async () => {
     setup({
       jsonData: {
         authType: AwsAuthType.ARN,
       },
     });
-    expect(screen.getByLabelText('Assume Role ARN')).toBeInTheDocument();
+    await waitFor(async () => expect(screen.getByLabelText('Assume Role ARN')).toBeInTheDocument());
```

Comment:
It because now there are async actions going on in the component (loading the data source). It we didn't add `waitFor`,, jest would make assertions before async actions were completed leading to falsy tests.